app/globals.css

```
@@ -77,14 +96,89 @@
+  html {
+    scroll-behavior: smooth;
+  }
+
   * {
     @apply border-border;
   }
+
   body {
-    @apply bg-background text-foreground;
+    @apply min-h-screen text-foreground antialiased;
+    font-feature-settings: "cv11", "ss01", "kern";
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)